packages/components/src/components/hds/app-footer/copyright.ts

```diff
@@ -5,7 +5,18 @@
 
 import Component from '@glimmer/component';
 
-export default class HdsAppFooterCopyrightComponent extends Component {
+export interface HdsAppFooterCopyrightSignature {
+  Args: {
+    year?: string;
+  };
+
+  Blocks: {
+    default: [];
+  };
```

Comment:
I don't think the template exposes blocks... 🤔
```suggestion
```

Reply:
Ok. I get this better now.